### DISCUSSION TITLE
Flying after the pandemic

### DISCUSSION THREAD

**Discussion by shmakes** [shmakes/hf-basic#209] [Ideas]
What extra information are hubs going to need to track when they start flying again due to Covid-19 concerns?

**Comment by bjbradl4** [comment]
We haven’t defined it yet.  We have a committee that will be meeting at the national level to determine what we will be requiring and what will be hub optional.  I know there is talk of vaccine as optional and covid test as required but they haven’t even had their first meeting yet. We are still waiting to hear back from legal.  Also if we are flying when it has been determined that there is herd immunity, will anything be required or will it all be optional?  Lots of questions so lots balls in the air. Nothing solid or definitive yet.

I can let you know as soon as we do.

Bobbie, COO
Honor Flight Network
4601 North Fairfax Drive Suite 1200
Arlington, Virginia 22203
(937) 521-2400 ext 104
[A close up of a logo  Description automatically generated]

**Comment by bjbradl4** [comment]
Good morning Steve

We now have our answer.  For 2021 They are required to be either vaccinated or a PCR Covid Test that needs to be verified through Clear Health Pass (CHP) .  So not sure if we want to have a check box for Vaccine, PCR Covid Test or Clear Health Pass or all three.    Or maybe it’s:


  *   CHP Vaccine
  *   CHP Covid Test

And would it be possible for them to be red flagged somehow if one of these are not checked? Your thoughts or what has Stars & Stripes suggested?



Bobbie, COO
Honor Flight Network
4601 North Fairfax Drive Suite 1200
Arlington, Virginia 22203
(937) 521-2400 ext 104
[A close up of a logo  Description automatically generated]

From: Steve Schmechel ***@***.***>
Sent: Saturday, January 30, 2021 1:47 PM
To: shmakes/hf-basic ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [shmakes/hf-basic] Flying after the pandemic (#209)


What extra information are hubs going to need to track when they start flying again due to Covid-19 concerns?
